hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -63,6 +63,9 @@
     private List<Note> notes;
 
     @JsonInclude(Include.NON_EMPTY)
+    private List<LifeCycle> lifecycle;
```

Comment:
For an alert is it true that alert.status == alert.lastLifecycle.state? (and the same for resolvedTime & resolvedBy)
Would it be easier to return those from the last alert instead of having to maintain them separately?